Zimbra/zm-mailbox
# Bugfix/zcs 7168

**Problem:** Move zimbraUseOwaspHtmlSanitizer from GlobalConfig to LocalConfig
**Fix:**
- Removed zimbraUseOwaspHtmlSanitizer from zimbra-attrs.xml
- Added zimbra_use_owasp_html_sanitizer in LC.java
- Updated DefangFactory to use LC.

**Testing done:**
- Tested manually
```
$zmlocalconfig zimbra_use_owasp_html_sanitizer
zimbra_use_owasp_html_sanitizer = true
```

**Testing to be done by QA:**
- Verify zimbraUseOwaspHtmlSanitizer is not available now on feature/owasp
- Verify zimbra_use_owasp_html_sanitizer is available on local config with default value "true"
- Verify defang functionality for changing value of zimbra_use_owasp_html_sanitizer.
